package.json

```diff
@@ -12,9 +12,18 @@
   "author": "",
```

Comment:
- [ ] (I couldn't be able to select the exact part for this bug, but is up from here) As I said in the last point, on the "scripts" section is important to have a start command like `"start": "webpack --config webpack.config.js --watch"`. Please add it. 🐱